src/config.sh

```diff
@@ -35,6 +35,11 @@ function config_main()
     exit 0
   fi
 
+  if [[ "$#" == 0 ]]; then
```

Comment:
It would be better if you have a check like this:
```
if [[ -n options_values['SHOW_CONFIG'] ]]
...
```

Reply:
Noticed now that you referred to the check right at the beginning of `config_main()`. I thought you where referring to the check in the beginning of `show_configurations()` 😭 
Fixing it ASAP.

Reply:
[UPDATE]:
I actually just moved this check from `config_main()` to `parse_config_options()` and unified the entrypoint to `show_configurations()` through a check inside `config_main()` with a check similar to the one suggested.